src/github.com/getlantern/fronted/direct.go

```diff
@@ -267,13 +292,13 @@ func (d *direct) headCheck(m *Masquerade) error {
 		Transport: trans,
 	}
 	url := "http://dlymairwlc89h.cloudfront.net/index.html"
-	if resp, err := client.Head(url); err != nil {
+	resp, err := client.Head(url)
+	if err != nil {
 		return err
-	} else {
-		defer resp.Body.Close()
-		if 200 != resp.StatusCode {
-			return fmt.Errorf("Unexpected response status: %v, %v", resp.StatusCode, resp.Status)
-		}
+	}
+	defer resp.Body.Close()
+	if 200 != resp.StatusCode {
+		return fmt.Errorf("Unexpected response status: %v, %v", resp.StatusCode, resp.Status)
```

Comment:
The above change to the http error handling doesn't change anything correct? Maybe unnecessary?